tests/acceptance/brocfile-smoke-test-slow.js

```diff
@@ -392,7 +393,7 @@ describe('Acceptance: brocfile-smoke-test', function() {
 
     return runCommand(path.join('.', 'node_modules', 'ember-cli', 'bin', 'ember'), 'build', '--silent')
       .then(function() {
-        var exists = fs.existsSync(path.join('.', 'dist', 'assets', appName + '.css'));
+        var exists = exists(path.join('.', 'dist', 'assets', appName + '.css'));
```

Comment:
Looks like another one here, but it won't matter when you've changed everything to use `existsSync` instead.